runtime/legion/legion_ops.cc

```diff
@@ -291,6 +300,20 @@ namespace Legion {
     void Operation::execute_dependence_analysis(void)
     //--------------------------------------------------------------------------
     {
+#ifdef DEBUG_LEGION
+      assert(!memoizing || trace != NULL);
+#endif
+      if (memoizing)
```

Comment:
Does this still belong here or should this be handled by the Memoizable trait now?